service/static/index.html

```diff
@@ -26,11 +26,12 @@ <h1>Manage wishlists</h1>
         <div class="d-inline-block text-wrap" id="error-message"></div>
         <button type="submit" class="d-inline-block btn btn-link" id="wishlists_clear">Clear all fields</button>
       </div>
-      <div class="table-responsive">
+      <-- <div class="table-responsive">
```

Comment:
This is not valid html. I'm not sure why you were trying to comment out the status message section, don't we need it?

Reply:
I have already added status part below this line.

Reply:
Is this not needed anymore then? Could you remove it then?